src/translator/cache.h

```diff
@@ -26,7 +27,13 @@ class AtomicCache {
 
   void store(const Key &key, Value value) { atomicStore(key, value); }
 
-  const Stats stats() const { return stats_; }
+  const Stats stats() const {
+#ifdef ENABLE_CACHE_STATS
+    return Stats{hits_.load(), misses_.load()};
+#else
+    return Stats{0, 0};
```

Comment:
How about adding an `std::cerr << "Warning: Cache stats disabled. To enable recompile with -DENABLE_CACHE_STATS=ON"<< std::endl;` (or use a logger, I think there is WARN_ONCE or something).

That way anyone who uses the `stats()` call when the stats are not tracked will know not to trust the result, as well as get a hint on how to enable them.

Reply:
If somebody calls this without enabling the same in build, it's converted into a fatal abort, which should serve the same purpose of informing the user.

Reply:
Not sure if it needs to be fatal, but as you decide. Merge when you feel like it